setup/install

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 
+cd api-server
+wget https://repo1.maven.org/maven2/org/jacoco/org.jacoco.agent/0.8.6/org.jacoco.agent-0.8.6-runtime.jar
```

Comment:
This jar should be downloaded only when vermillion is being tested

Reply:
Made changes